covenant-signer/README.md

```diff
@@ -0,0 +1,301 @@
+# Covenant Signer
```

Comment:
Phase-1 covenants have also used the Phase-1 "Covenant Signer" (github.com/babylonlabs-io/covenant-signer), which is a completely different program than this. To avoid confusion, maybe we should add some small letters note here that this is a different program.